core/templates/pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.component.spec.ts

```diff
@@ -56,7 +56,7 @@ describe('Subtopic viewer navbar breadcrumb component', function() {
         {
           provide: SubtopicViewerBackendApiService,
           useValue: {
-            fetchSubtopicData: () => (
+            fetchSubtopicData: async() => (
```

Comment:
Also add Async at the end of the function name (and edit the callsites)

Reply:
I was adviced that I can ignore changing the function name as I am working in a spec file.

Reply:
Hi @Bhawana15, let's rename `fetchSubtopicData` to `fetchSubtopicDataAsync`. Do a similar change for `fetchTotalQuestionCountForSkillIds`.

Reply:
Okay Thanks @kevintab95